src/http/representation/RepresentationMetadata.ts

```diff
@@ -415,4 +419,17 @@ export class RepresentationMetadata {
       this.set(CONTENT_LENGTH_TERM, toLiteral(input, XSD.terms.integer));
     }
   }
+
+  /**
+   * Converts the incoming name (URI or shorthand) to a named node.
+   * The generated terms get cached to reduce the number of created nodes,
+   * so only use this for internal constants!
+   * @param name - Predicate to potentially transform.
+   */
+  private toCachedNamedNode(name: string): NamedNode {
```

Comment:
Would make this a utility function above the class description instead of being part of the class.